backend/penndata/analytics.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
```

Comment:
wasn't sure if it was good practice to use the pennmobile `analytics.py` file for penndata analytics calls (though they're essentially the same file.)